### ISSUE TITLE
[usdview] Layered alembic transform payloads not always working

### ISSUE THREAD

**Issue by AlexSchwank** [PixarAnimationStudios/USD#1118]
### Description of Issue
I'm running into an issue when I use multiple alembic payloads for a prim for transformations.
I have two files: _teapot.abc_ containing a teapot mesh and _teapotTransform.abc_ containing a transform for the teapot. Layering them using payloads is causing issues in usdview (see Case  2).

**Case 1: no layers**
```
def Xform "myTeapot_original" {
    over "teapot_teapot" (
        payload = [
            @teapot.abc@</teapot_teapot>
        ]
    )
    {}
}
```
A non-transformed teapot. No surprise here.

![teapot_original](https://user-images.githubusercontent.com/1038475/74830892-082b7480-530c-11ea-9279-d2c1866f5fb0.png)

**Case 2: abc transform (error case)**
```
def Xform "myTeapot_abcTransform"
{
    over "teapot_teapot" (
        payload = [
            @teapotTransform.abc@</teapot_teapot>,
            @teapot.abc@</teapot_teapot>
        ]
    )
    {}
}
```
This is the case where it doesn't work. The teapot is _not_ transformed. However, `usdview` is showing the transform attributes for _teapot_teapot_. 

![teapot_abcTransform](https://user-images.githubusercontent.com/1038475/74830886-01046680-530c-11ea-9fbc-ef9eaf1bcc83.png)

**Case 3: usd transform**
```
def Xform "myTeapot_usdTransform"
{
    over "teapot_teapot" (
        payload = [
            @teapotTransform.usda@</teapot_teapot>,
            @teapot.abc@</teapot_teapot>
        ]
    )
    {}
}
```
This works as expected. The teapot is transformed. _teapotTransform.usda_ was generated by `usdcat teapotTransform.abc -o teapotTransform.usda`. `usdview` is showing the same transform attributes for _teapot_teapot_ as in Case 2.

![teapot_usdTransform](https://user-images.githubusercontent.com/1038475/74830861-f649d180-530b-11ea-937f-c4b0a2944d83.png)

**Case 4: flattened stage**
```
def Scope "Assets"
{
    def Xform "myTeapot_usdTransform"
    {
        def Xform "teapot_teapot"
        {
            float3 xformOp:rotateZXY = (1, 2, 3)
            float3 xformOp:scale = (1.1, 1.2, 1.3)
            double3 xformOp:translate = (2, 3, 4)
            uniform token[] xformOpOrder = ["xformOp:translate", "xformOp:rotateZXY", "xformOp:scale"]

            def Mesh "teapot_teapotShape"
            {
                #...
            }
        }
    }

    def Xform "myTeapot_abcTransform"
    {
        def Xform "teapot_teapot"
        {
            float3 xformOp:rotateZXY = (1, 2, 3)
            float3 xformOp:scale = (1.1, 1.2, 1.3)
            double3 xformOp:translate = (2, 3, 4)
            uniform token[] xformOpOrder = ["xformOp:translate", "xformOp:rotateZXY", "xformOp:scale"]

            def Mesh "teapot_teapotShape"
            {
                #...
            }
        }
    }

    def Xform "myTeapot_original"
    {
        def Xform "teapot_teapot"
        {
            def Mesh "teapot_teapotShape"
            {
                #...
            }
        }
    }
}
```
This works as well. Both _myTeapot_usdTransform_ and _myTeapot_abcTransform_ are transformed correctly.

[teapot_files.zip](https://github.com/PixarAnimationStudios/USD/files/4224822/teapot_files.zip)

### Package Versions
Tested in 19.07 and 19.11

**Comment by spiffmon** [comment]
Hi @AlexSchwank !
    The behavior you are describing sounds incredibly weird.  Unfortunately, the transform values in your example are really subtle and I am having a hard time seeing any difference in the three pictures you provided...  If you can reword it to throw in a 90 degree rotation or a larger scale, I can at least try to verify the behavior here.  Honestly, if we see the same behavior I think it’s unlikely we’d be able to investigate anytime soon... this is a rather unexpected use for alembic files in USD (though I can’t imagine how it would screw up if usdcat gives the right answer).  Can I point you at. PR #1099 , which we are hoping to accept soon, which would allow you to use ALembic’s own layering feature to compose your alembic files in Alembic-space, which might give you bettter ransform comPosing, since we flatten out all the Alembic xform ops?

**Comment by AlexSchwank** [comment]
Hi @spiffmon ,

The teapot in the top right corner is the transformed one. The teapot at the origin has not transform.
I'll check out the PR - that might solve some problems on our end.

**Comment by jilliene** [comment]
Filed as internal issue #USD-5879

**Comment by spitzak** [comment]
Isn't usd ignoring the root transform of abc files? Or am I thinking of
something in-house here
